resources/assets/coffee/react/_components/beatmap-basic-stats.coffee

```diff
@@ -0,0 +1,46 @@
+###
+# Copyright 2015-2016 ppy Pty. Ltd.
+#
+# This file is part of osu!web. osu!web is distributed with the hope of
+# attracting more community contributions to the core ecosystem of osu!.
+#
+# osu!web is free software: you can redistribute it and/or modify
+# it under the terms of the Affero GNU General Public License version 3
+# as published by the Free Software Foundation.
+#
+# osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+###
+{div, span} = React.DOM
+
+bn = 'beatmap-basic-stats'
+
+@BeatmapBasicStats = ({beatmapset, beatmap}) ->
+  div
+    className: bn
+    for stat in ['total_length', 'bpm', 'count_circles', 'count_sliders']
```

Comment:
>`count_circles` and `count_sliders` are not implemented yet

it's not that they are not implemented, these just don't need to be altered in any way before being displayed

also, [this](https://github.com/ppy/osu-web/blob/master/app/Transformers/BeatmapTransformer.php)